Source/Mosa.DeviceSystem/Services/PCIDeviceService.cs

```diff
@@ -62,9 +64,22 @@ private void CreatePCIDevice(byte bus, byte slot, byte function, Device device,
 			matchPriority = priority;
 		}
 
-		// No driver found
-		if (matchedDriver == null)
-			return;
+		// Set a dummy entry if none were found
+		matchedDriver ??= new PCIDeviceDriverRegistryEntry
```

Comment:
It's not appropriate to create dummy driver entries for unknown devices. A PCIDevice is already registered with the device system so that is enough to know that the device exists

Reply:
The idea is to be able to get every PCI device without relying on that specific class. The thing is, PCIDevice is not really a PCI device by itself, but rather a helper class for dealing with generic PCI stuff (which is why I personally want to not make it inherit from BaseDeviceDriver anymore, thus getting rid of that layer in the device driver tree). Registering a dummy entry allows for getting the PCI device drivers at the lowest level (which is themselves), instead of having to go through a PCIDevice, and getting its children.

Reply:
Wouldn't it be better to make the base Device class BusType aware instead? That way you remove dependency on the driver entry for checking if a device is from a particular bus type, no longer needing dummy driver entries.